kodilla-basic-tests/src/test/java/com/kodilla/homework/BankTestSuite.java

```diff
@@ -0,0 +1,177 @@
+package com.kodilla.homework;
+
+import com.kodilla.bank.homework.Bank;
+import com.kodilla.bank.homework.CashMachine;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BankTestSuite {
+
+    @Test
+    public void shouldWorkWhenBankHasNoCashMachine() {
+        //given
+        Bank bank = new Bank(new CashMachine[0]);
+
+        //when then
+        assertEquals(0, bank.getTotalBalance());
+        assertEquals(0, bank.getTotalNumberOfDeposits());
+        assertEquals(0, bank.getTotalNumberOfWithdrawals());
+        assertEquals(0, bank.getTotalAverageOfDeposits());
+        assertEquals(0, bank.getTotalAverageOfWithdrawals());
+    }
+
+    @Test
+    public void shouldWorkWhenBankHasOneEmptyCashMachine() {
+        //given
+        CashMachine a1 = new CashMachine();
+        CashMachine a2 = new CashMachine();
+        CashMachine a3 = new CashMachine();
+
+        a1.add(200);
+        a1.add(-100);
+        a2.add(500);
+        a2.add(-100);
+
+        Bank bank = new Bank(new CashMachine[] {a1, a2, a3});
+
+        //when then
+        assertEquals(500, bank.getTotalBalance());
+        assertEquals(2, bank.getTotalNumberOfDeposits());
+        assertEquals(2, bank.getTotalNumberOfWithdrawals());
+        assertEquals(350, bank.getTotalAverageOfDeposits());
```

Comment:
Wychodza nam liczby ujemne :) powinny byc wszystkie dodatnie :) Musisz zmieniac znak w metidach w CashMachine

Reply:
Średnia kwota wypłat powinna być dodatnia ? Jak to ? 
Czy chodzi o zmiane sumOfWithDrawals "+=" na "-=" i wtedy uzywać w testach samych liczb dodatnich ?

Reply:
Jak cie spytam, ile wyplaciles z bankomatu, to nie powiesz mi -200 tylko 200. Wiec metoda, ktora liczy sume wyplat powinna zrobic return -suma

Reply:
No tak, logiczne :)